master/buildbot/www/rest.py

```diff
@@ -249,6 +249,15 @@ def checkFields(fields, negOk=False):
                 except Exception:
                     raise BadRequest('invalid offset')
                 continue
+            # As discussed with @tardyp, this is the best way to implement
+            # properties behavior.
+            elif arg in ('property',):
+                try:
+                    properties = [v.decode('utf-8') for v in reqArgs[arg]]
+                except Exception:
+                    raise BadRequest('invalid property value for %s' % arg)
+                filters.append(resultspec.Filter(arg, 'eq', properties))
```

Comment:
How to handle ``field`` options ?

**What should happen for those use cases**:

+ `api/v2/builds/1?field=buildid&property=slavename`
+ `api/v2/builds?field=buildid&property=slavename`

knowing that field ``property`` does not exists in the Builds table as already mentioned ??